src/docfx.website.themes/pdf.default/bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "pdf.default",
+  "description": "This is a docfx template to generate pdf files",
+  "authors": [
+    "docfx"
+  ],
+  "license": "MIT",
+  "keywords": [
+    "docfx",
+    "template"
+  ],
+  "homepage": "https://github.com/dotnet/docfx",
+  "moduleType": [],
+  "private": true,
```

Comment:
private means our package will not publish to bower https://github.com/bower/spec/blob/master/json.md#private